packages/react-native/React/CoreModules/RCTRedBox.mm

```diff
@@ -70,33 +70,40 @@ - (void)loadExtraDataViewController;
 
 @end
 
-@interface RCTRedBoxWindow : NSObject <UITableViewDelegate, UITableViewDataSource>
-@property (nonatomic, strong) UIViewController *rootViewController;
+@interface RCTRedBoxWindow : UIViewController <UITableViewDelegate, UITableViewDataSource>
```

Comment:
This class (and the instance variables) need to be renamed, since this is no longer a UIWindow subclass (iirc it was one in the past). Could you make this `RCTRedBoxController`?

Reply:
Good catch! I've renamed it and all method params that referenced `redBoxWindow`